Sources/MUXSDKStatsObjc/MUXSDKStats.m

```diff
@@ -20,206 +14,19 @@
 
 @implementation MUXSDKStats
 
-static MUXSDKDispatcher *_dispatcher;
```

Comment:
This `_dispatcher` was unused, so removed it completely.

Reply:
That includes the `MUXSDK*DataStore` references, these are now held by `MUXSDKPlayerBindingManager`.